commons-numbers-core/src/main/java/org/apache/commons/numbers/core/ArithmeticUtils.java

```diff
@@ -58,6 +58,8 @@ private ArithmeticUtils() {
      *   for the special cases above.</li>
      *  <li>The invocation {@code gcd(0, 0)} is the only one which returns
      *   {@code 0}.</li>
+     *   <li>gcd can be used to determine if two numbers are relatively prime.
```

Comment:
This has been added as a special case. It is not a special case, it is a usage example. It should be moved to a new paragraph, e.g.
```
<p>Two numbers are relatively prime, or coprime, if their gcd is 1.
```

Reply:
@aherbert fixed JavaDoc